### DISCUSSION TITLE
Repository improvements

### DISCUSSION THREAD

**Discussion by jputcu** [ETLCPP/etl#778] [Ideas]
Hello,

A few questions:

- Is `support/avr-stl-1.1.1.zip` still being used? It is taking up almost 200kb?
- Wouldn't it be better to replace `support/Release notes.txt` with a more conventional `CHANGELOG.txt/md/...` in the root?
- Perhaps a hint to a nice changelog format https://keepachangelog.com/en/1.0.0/

**Comment by jwellbelove** [comment]
Sorry for the delayed response! I don't get any notifications to new discussion topic!
I'll take a look at your suggestions.

**Comment by jwellbelove** [comment]
I've removed the AVR STL zip file.
